Github add-labels script references non-existent `ping-codeowners.sh` script

### Component(s)

_No response_

### Describe the issue you're reporting

When attempting to add a component label to an open issue by leaving a comment, the `add-labels.sh` [script](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/3e6c2a6430a9a0c35d7a2b58eb1727cfd423dfb3/.github/workflows/scripts/add-labels.sh) fails trying to ping the codeowners of the component.

[Failure reference](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/6040791721/job/16392379955)

Failure output:
```
Run ./.github/workflows/scripts/add-labels.sh
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/2453[9](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/runs/6040791721/job/16392379955#step:4:10)
bash: ./.github/workflows/scripts/ping-codeowners.sh: No such file or directory
Error: Process completed with exit code 127.
```

Line failing is found [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/3e6c2a6430a9a0c35d7a2b58eb1727cfd423dfb3/.github/workflows/scripts/add-labels.sh#L62).
```
        # Labels added by a GitHub Actions workflow don't trigger other workflows
        # by design, so we have to manually ping code owners here.
        COMPONENT="${LABEL}" ISSUE=${ISSUE} SENDER="${SENDER}" bash "${CUR_DIRECTORY}/ping-codeowners.sh"
```
There is no `ping-codeowners.sh` file, but there are `ping-codeowners-issues.sh`, `ping-codeowners-on-new-issue.sh`, and `ping-codeowners-prs.sh` in this directory.

I believe `ping-codeowners.sh` should be updated to be `ping-codeowners-issues.sh`, but I'd have to test to make sure.

--- a/.github/workflows/scripts/add-labels.sh
+++ b/.github/workflows/scripts/add-labels.sh
@@ -59,7 +59,7 @@ for LABEL_REQ in ${LABELS}; do
 
         # Labels added by a GitHub Actions workflow don't trigger other workflows
         # by design, so we have to manually ping code owners here.
-        COMPONENT="${LABEL}" ISSUE=${ISSUE} SENDER="${SENDER}" bash "${CUR_DIRECTORY}/ping-codeowners.sh"
+        COMPONENT="${LABEL}" ISSUE=${ISSUE} SENDER="${SENDER}" bash "${CUR_DIRECTORY}/ping-codeowners-issues.sh"
     else
         gh issue edit "${ISSUE}" --remove-label "${LABEL}"
     fi

